packages/web/src/components/track/PremiumTrackSection.tsx

```diff
@@ -224,10 +243,11 @@ const LockedPremiumTrackSection = ({
     if (premiumConditions.nft_collection) {
       return (
         <Button
+          color='accentBlue'
```

Comment:
sry did not check out your other PR for passing colors down to stems components, just double checking that this is what we wanna do rather than passing `palette.accentBlue`?

Reply:
I don't think `palette` is a thing in web, since we use CSS variables for colors. This is about as close as we can get.
Though I think if we wanted to unify it on mobile, we would do a similar approach, but the allowable values will be any of the property names that exist on the `palette` object.